includes/models/model.llms.user.achievement.php

```diff
@@ -15,16 +15,17 @@
  *
  * @since 3.8.0
  * @since 3.18.0 Unknown.
+ * @since [version] Declare `achievement_title` and `achievement_content` properties.
```

Comment:
The only merge conflicts that are apparent with your branch and mine are with this file. I've recorded some todos on my branch related to these two fields. I think your changes can be discarded and we can figure out a better way to manage this after the merge.

Reply:
Fixed...
Are you fine with me rebasing (not merging) my branch against yours and working on it?

Reply:
That's fine with me, I was planning on merging your branch into mine though so that I can review what you're working on more thoroughly and so that we can start to sync up

Reply:
Alright, rebased and force pushed.